source/styles/home.styl

```diff
@@ -0,0 +1,57 @@
+.home-slogan
+  text-align center
+
+.home-description
+  text-align center
+  margin-bottom 40px
+
+.home-download
+  background: #eee
+  display: flex
+  justify-content space-between
+  align-items center
+
+  a
+    text-decoration none
+
+.home-logo
+  padding 20px 40px 20px 20px
+  img
+    height 120px
+
+.home-download-details
+  flex 1
+  p
+    margin 0
+    padding 0
+
+  .home-download-version
+    display block
+    font-size 1.4rem
+    font-weight 700
+    color black
+    border-radius 0 4px 0 0
+    margin-bottom 20px
+
+  .home-download-list
+    color rgba(0, 0, 0, 0.5)
+    font-size 0.9rem
+    padding-top 0
+    padding-bottom 0
+    height 28px
+    max-height 28px
+
+    a:after
+      content ', '
+
+    a:nth-last-child(2):after
+      content ' & '
+      color rgba(0, 0, 0, 0.5)
+
+    a:last-child:after
+      content ''
+
+.home-secondary-links
+  margin-top 40px
+  p
+    text-align center
```

Comment:
Yup! In fact, likely other volunteers could handle this. I did a sample localization myself. It is pretty easy (just time consuming) to copy paste from the Markdown to the new dictionary definitions. But you can tell what-is-what on the homepage.